### PR TITLE
Feat/stopewatch

### PR DESCRIPTION
**Pull request by gillsheenu**

### CODE REVIEW

File: app/src/main/java/com/example/nickelffoxassignments_sheenu/MainActivity.kt [hunk 1]
```diff
@@ -85,8 +87,9 @@ class MainActivity : AppCompatActivity() {
                 }
                 R.id.stopWatchFragment -> {
                     setColorEnabled = true
-                    setAppBarColor(R.color.stopWatch_color)
+                    setAppBarColor(R.color.new_stop_watch_color)
```
Comment (adhish-nickelfox) [inline app/src/main/java/com/example/nickelffoxassignments_sheenu/MainActivity.kt:89]: Color name should be readable

File: app/src/main/res/layout/fragment_stop_watch.xml [hunk 2]
```diff
@@ -35,30 +35,113 @@
...
+                android:textColor="#FFFFFF"
+                android:text="LAP"
+                app:backgroundTint="#202020" />
+
+        </androidx.cardview.widget.CardView>
+
+
+
+
     </LinearLayout>
```
Comment (adhish-nickelfox) [inline app/src/main/res/layout/fragment_stop_watch.xml:170]: Use constraint layout instead of Linear Layout @gillsheenu

**Review by adhish-nickelfox — CHANGES_REQUESTED** [review]

**Review by pratikshit-nickelfox — COMMENTED** [review]
stop_watch_gradient.xml is not being used and is empty. It should be removed.

**Review by adhish-nickelfox — APPROVED** [review]
LGTM

**Comment by pratikshit-nickelfox** [comment]
All changes reviewed. Ready to merge.

**Pull request merged**